lib/client-assets.php

```diff
@@ -297,8 +297,16 @@ function gutenberg_register_scripts_and_styles() {
 			array(
 				'( function() {',
 				'	if ( wp && wp.components && wp.serverSideRender && ! wp.components.ServerSideRender ) {',
```

Comment:
Doesn't the last part of this condition become unnecessary, or at least defeat the purpose of adding a deprecation? In other words, should we override `wp.components.ServerSideRender` even if it already exists?

Reply:
I guess it is safe to override wp.components.ServerSideRender even if it already exists, the code was updated.